scripts/generate_metadata.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+import subprocess
+
+AIRPLANE_METADATA_FILENAME = "metadata.json"
+AIRPLANE_METADATA_FOLDER = ".metadata"
+
+REQUIRED_KEYS = ["name"]
+
+def generate_metadata():
+    print("Generating metadata...")
+
+    root = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../")
+
+    all_metadata = {}
+    for path in os.listdir(root):
+        metadata_folder = os.path.join(root, path, AIRPLANE_METADATA_FOLDER)
+        if not os.path.isdir(os.path.join(root, path)):
+            continue
+        if not os.path.isdir(metadata_folder):
+            continue
```

Comment:
```suggestion
        if not os.path.isdir(metadata_folder):
            continue
```

Do we need the first check?